### DISCUSSION TITLE
Few QOL Easy/Quick to add Suggestions

### DISCUSSION THREAD

**Discussion by Chromaah** [diamond-rip/Eden#355] [Ideas]
1 - **Match Start Countdown Title**

Add a title that shows the countdown of the match starting. Therefore, it won't be only in the chat and will look much cleaner. Also adding the title that says 'Match Started!' or 'Go!' after the countdown ends.
**-------------------------------------------------------------------------------------------**
2 - **Bridge/Other Kit Team Color/Indicator**

Begging for this one lol. I've explained it in an old suggestion issue, but I believe it got dug up and never considered. Here's the repost:

- Kits that have Point System, portal goal, bed, etc. (any option that adds teams) enabled, like bridge, battle rush, bed fight, etc have death messages according to the team color instead of enemies.

- For example, instead of &aChromaah killed &cEnemy in games with point system, portal goal, bed, etc. (any option that adds teams) have it be the color of their team:

>         &cChromaah killed &9Enemy
>         (If Chromaah was on red team, and Enemy was on blue team)
**-------------------------------------------------------------------------------------------**
3 - **Portal Goal Enter**

Make it so that in the last goal you need to win, instead of leaving the player in the portal, teleport them back to their spawn point, it will be way cleaner IMO.
**-------------------------------------------------------------------------------------------**
4 - **Golden Apples in Bridges**

I believe this is a bug, but Golden Apples when playing bridges for example, don't give absorption. (Instant Golden Apple kit rule is enabled)

**Comment by GoodestEnglish** [comment]
About 4, set `match.instant-gapple-effects` to true

**Reply by GoodestEnglish** [reply]
I will look others after the bugs in issues.

**Reply by GoodestEnglish** [reply]
Adding point 1 in next update
